src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
It's in the messagingReadyPromise, so it will never be queued.  it's exactly the same to call either `sendMessageUnreliable_` or `messageDeliverer_` here.